prime-router/metadata/valuesets/common.valuesets

```diff
@@ -19,7 +19,7 @@
   values:
   - code: A
     display: Abnormal (applies to non-numeric results)
-  - code: ">"
+  - code: "&#62;"
```

Comment:
I verified with ./prime generate-docs and it works well.  I believe that is only command this change would affect.

Reply:
I added the unit test case for checking the special code generation of '>'.

Reply:
The code '>' is now handling in the DocumentationFactery.kt instead of in common.valuesets schema.  Doing this will fix only during generating markdown documents.  It will not affect the output of the fake data generation.